README.md

```diff
@@ -254,15 +254,15 @@ import {
     DdRum
 } from '@datadog/mobile-react-native';
 
-// Start a view with a unique view identifier, a custom view url, and an object to attach additional attributes to the view
+// Start a view with a unique view identifier, a custom view name, and an object to attach additional attributes to the view
 DdRum.startView(
-    '<view-key>',
-    '/view/url',
+    'ViewName-with-unique-id',
```

Comment:
Shouldn't we align it with Android/iOS SDK, which is using `view key` + `view name`? I find `ViewName-with-unique-id` a bit confusing when we already have `View Name`.

Reply:
I based this on what is done for the react-navigation integration.

What would be a good `view key` in this case? We can put the main component name as `view name`, but I think there is no concept that could be used as the `view key` in React Native

Reply:
it can be anything, any unique identifier for the view, not necessary containing the view name itself. It has view name just because `react-navigation` library provides component ID in such a way, but in general for the manual instrumentation in can be anything.